src/managed_scip.jl

```diff
@@ -0,0 +1,74 @@
+# holds pointers to SCIP data and takes care of memory mgmt
```

Comment:
Nit: We prefer complete sentences with correct capitalization and periods at the end for comments.

Reply:
Done (for existing docstrings).